src/Lower.cpp

```diff
@@ -100,6 +102,19 @@ Stmt lower(vector<Function> outputs, const string &pipeline_name, const Target &
         debug(1) << "Skipping injecting memoization...\n";
     }
 
+    size_t no_prefetch;
+    get_env_variable("HL_NO_PREFETCH", no_prefetch);
```

Comment:
I will remove it...I was using it for a quick way to compare with and without prefetch without changing the source of the tests.